lib/mspec/runner/actions/constants_leak_checker.rb

```diff
@@ -0,0 +1,86 @@
+class ConstantsLeakChecker
+  attr_accessor :constants_new
+
+  def initialize
+    @constants_before = Module.constants
+  end
+
+  def check
+    @constants_after = Module.constants
+    @constants_new = @constants_after - @constants_before
+    @constants_new.empty?
+  end
+end
+
+class ConstantsLockFile
+  LOCK_FILE_NAME = '.mspec.constants.lock'
+
+  def self.exist?
+    File.exist?(LOCK_FILE_NAME)
+  end
+
+  def self.load
+    File.read(LOCK_FILE_NAME).split
```

Comment:
Any reason to not use `File.readlines` here?
Also I'd make load return `[]` if the file doesn't exist, then calling code can be simplified quite a bit.